File: Python/Programmers/두정수사이의합.py
def solution(a,b):
    # if a <= b:
    #     min = a
    #     max = b
    # else:
    #     min = b
    #     max = a
    if a>b:
        a,b = b,a #default가 a<b라서 아닌 경우만 바꿔주기
    return sum(range(a, b+1))
    # count = min
    # answer = min
    # while True:
    #     if count == max:
    #         break
    #     count += 1
    #     answer += count
    # return answer

print(solution(3,5))
print(solution(3,3))
print(solution(5,3))

File: Java/Programmers/두정수사이의합.java

public class 두정수사이의합 {
	public static void main(String args[]) {
		Solution두정수사이의합 s = new Solution두정수사이의합();
		System.out.println(s.solution(3, 3));
	}
}

class Solution두정수사이의합 {
	public long solution(int a, int b) {
		long answer = 0;
		for(int i=(a < b ? a : b); i<=(a < b ? b : a); i++) {
			answer += i;
		}
		return answer;
	}
}